### ISSUE TITLE
chromium: update to 112.0.5615.49

### ISSUE THREAD

**Issue by MaxIhlenfeldt** [OSSystems/meta-browser#704]
Release notes: https://chromereleases.googleblog.com/2023/04/stable-channel-update-for-desktop.html

**Comment by MaxIhlenfeldt** [comment]
Not sure if I can do the update this week, but I should be able to devote some time next week.

**Comment by nrpt-m** [comment]
Great, I can do some testing next week when the update is available like @rwmacleod usually does. 
Meanwhile, I have an update to 111.0.5563.147 version that fixes a critical CVE: https://nvd.nist.gov/vuln/detail/CVE-2023-1529
The update was simple, no patch adjustment so, I am testing it now.
Here it is: https://github.com/nrpt-m/meta-browser/commit/0eda2bb89200e4cd7a768f605d0e6a0ee55e2775

**Comment by MaxIhlenfeldt** [comment]
Short status update: I think I've gotten langdale and kirkstone to compile (there was a build fix that needed to be backported), but dunfell is still not building even after already fixing two errors.

**Comment by rwmacleod** [comment]
What sort of error are you seeing?

**Comment by MaxIhlenfeldt** [comment]
This is the complaint:
```
| ../../cc/paint/paint_op_writer.h:107:27: warning: inline function 'cc::PaintOpWriter::SerializedSize<unsigned int>' is not defined [-Wundefined-inline]
|   static constexpr size_t SerializedSize();
|                           ^
| ../../cc/paint/paint_op_writer.h:143:42: note: used here
|     return (base::CheckedNumeric<size_t>(SerializedSize<size_t>()) +
|                                          ^
```
I think clang 12 doesn't like https://crrev.com/c/4239982, I'm currently doing another build with that reverted to see if it's really the cause.

**Comment by MaxIhlenfeldt** [comment]
Turned out that was in fact just a warning and not an error, the actual culprit is that clang 12 doesn't implement [P2085R0](https://wg21.link/P2085R0), but clang 14 does, and there's some upstream code using that. There will probably be more uses of it in future releases, I already spotted one that will be part of 114.

I'll get the PR ready on Monday, have a nice weekend everyone!

**Comment by MaxIhlenfeldt** [comment]
Closed by #709.
